<launch>
  <!--  delay estimator -->
  <arg name="delay_estimator_param" default="$(find-pkg-share delay_estimator)/config/delay_estimator.yaml"/>
  <group>
    <push-ros-namespace namespace="calibration"/>
    <node pkg="delay_estimator" exec="delay_estimator" name="delay_estimator" output="screen" respawn="true">
      <param from="$(var delay_estimator_param)"/>
      <param name="node_name" value="steer"/>
      <!--input topic info-->
      <param name="input_type" value="autoware_auto_control_msgs/msg/AckermannControlCommand"/>
      <param name="input_access" value="lateral.steering_tire_angle"/>
      <param name="input_topic" value="/control/command/control_cmd"/>
      <!--response topic info-->
      <param name="response_type" value="autoware_auto_vehicle_msgs/msg/SteeringReport"/>
      <param name="response_access" value="steering_tire_angle"/>
      <param name="response_topic" value="/vehicle/status/steering_status"/>
    </node>
  </group>
</launch>
